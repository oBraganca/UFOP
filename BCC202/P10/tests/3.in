3
20 54 9 -10 -23 -17 -15 -27 17 -2 25 0 24 1 -20 23 -14 -16 -22 8 16 21 -7 -24 19 2 4 12 15 3 6 11 -11 -13 -6 -4 18 22 -26 -21 -9 26 -8 -1 -25 7 -3 -18 5 20 13 -5 14 -12 -19 10
-28 39 -2 -6 10 -8 -17 -1 -4 -13 -15 12 5 4 -3 8 -14 -5 -18 18 -11 -16 -7 7 14 -10 -12 0 13 3 1 9 17 16 11 2 -9 6 -19 15 19
-63 75 -29 -8 21 10 -37 29 -9 -1 -15 -18 -10 2 34 -19 -7 17 19 0 -11 -5 3 -31 11 9 -12 -22 -34 -17 -2 -20 4 -16 -13 -33 -28 -24 20 1 36 32 27 -6 -30 6 14 16 23 8 18 33 25 -25 -4 -14 -32 -35 31 -21 35 26 7 37 24 13 12 28 -26 30 -27 -3 5 15 22 -23 -36
