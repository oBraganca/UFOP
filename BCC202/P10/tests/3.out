48 3
39 5
75 6
